firmware/applications/SignBuddy/board.h

```diff
@@ -46,3 +47,35 @@
 #endif // defined(BOARD_DISCO)
 
 #endif // BOARD_H
+
+/* UART */
```

Comment:
let's split everything into two `board.h` files: `nucleo_board.h` and `disco_board.h` then `board.h` can just import whichever it needs depending on the define

Reply:
I agree with this